addons/web/static/tests/legacy/fields/relational_fields/field_one2many_tests.js

```diff
@@ -10092,6 +10092,45 @@ QUnit.module('fields', {}, function () {
             form.destroy();
             assert.verifySteps(["willUnmount 1"]);
         });
+
+        QUnit.test('o2m with sub view, with <create> tag and common default_* key in the context: ensure <create> value is the one kept', async function (assert) {
```

Comment:
name is really long, what about something like: "combine contexts on one2many field and create tags"